apps/api/package.json

```diff
@@ -96,9 +96,11 @@
     "ioredis": "^5.2.4",
     "jsonwebtoken": "9.0.0",
     "lodash": "^4.17.15",
+    "logform": "^2.5.1",
     "nanoid": "^3.1.20",
     "nest-raven": "8.2.0",
     "newrelic": "9.10.0",
+    "nest-winston": "^1.8.0",
```

Comment:
A small ideation here, did you had a chance to check out https://github.com/pinojs/pino ? Seems like the also have the same easy integration with NewRelic but have a much lower performance impact on the application running it and comes with a built in interceptor for HTTP requests aswell. Wdyt?